main.go

```diff
@@ -510,6 +515,12 @@ func main() {
 		if _, err = github.NewRepositoryRuleset(ctx, "nomad-server-default", &nomadServerDefaultRepositoryRulesetArgs); err != nil {
 			return err
 		}
+		if err = AddGithubTokenSecret(ctx, conf, "nomad-server"); err != nil {
```

Comment:
Could this container "User", which is the term that distinguishes the two tokens? The other function has admin so that'd made these a nice pair